modules/squashfuse/0.5.2/presubmit.yml

```diff
@@ -0,0 +1,8 @@
+matrix:
+  platform: ["ubuntu2204"]
+  bazel: ["6.x", "7.x"]
+tasks:
+  verify_targets:
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets: [//...]
```

Comment:
This should be `@squashfuse//...`, but it probably only works with 7.x since 6.x doesn't support overlay in `source.json`.

Reply:
Thanks, fixed. also removed macos, same reason as with libfuse this is only really a Linux thing.